packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -1,6 +1,45 @@
 ---
 title: Okta API Products Release Notes 2021
 ---
+# September
+
+### Monthly Release 2021.09.0
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [SAML assertion preview populates with real data](#saml-assertion-preview-populates-with-real-data)              | September 1, 2021 |
+| [Dynamic Issuer Mode is EA in Preview](#dynamic-issuer-mode-is-ea-in-preview) | September 1, 2021 |
+| [The Subscriptions API is GA in Production](#the-subscriptions-api-is-ga-in-production)| September 1, 2021 |
+| [Rate limit updates](#rate-limit-updates)| September 1, 2021 |
+
+#### SAML assertion preview populates with real data
+
+After a developer creates a SAML App integration in the Okta Admin Console, the preview of a generated SAML assertion now uses real data instead of mock data. See [Create a SAML integration](/docs/guides/build-sso-integration/saml2/create-you). <!--OKTA-398842-->
```

Comment:
```suggestion
After a developer creates a SAML App integration in the Okta Admin Console, the preview of a generated SAML assertion now uses real data instead of mock data. See [Create a SAML integration](/docs/guides/build-sso-integration/saml2/create-your-app/). <!--OKTA-398842-->
```